patches/minecraft/net/minecraft/client/renderer/texture/LayeredColorMaskTexture.java.patch

```diff
@@ -0,0 +1,27 @@
+--- ../src-base/minecraft/net/minecraft/client/renderer/texture/LayeredColorMaskTexture.java
++++ ../src-work/minecraft/net/minecraft/client/renderer/texture/LayeredColorMaskTexture.java
+@@ -71,9 +71,15 @@
+ 
+                         if (s != null)
+                         {
++                            BufferedImage bufferedimageTemp = net.minecraftforge.event.ForgeEventFactory.getResourceBufferImage(s);
++                            if (bufferedimageTemp == null)
++                            {
+                             iresource1 = p_110551_1_.func_110536_a(new ResourceLocation(s));
+                             BufferedImage bufferedimage2 = TextureUtil.func_177053_a(iresource1.func_110527_b());
+-
++                            bufferedimageTemp = bufferedimage2;
++                            }
++                            BufferedImage bufferedimage2 = bufferedimageTemp;
++                            
+                             if (bufferedimage2.getWidth() == bufferedimage.getWidth() && bufferedimage2.getHeight() == bufferedimage.getHeight() && bufferedimage2.getType() == 6)
+                             {
+                                 for (int l = 0; l < bufferedimage2.getHeight(); ++l)
+@@ -98,6 +104,7 @@
+                     }
+                     finally
+                     {
++                        if (iresource1 != null)
```

Comment:
This patch isn't needed, `closeQuietly` null-checks already.

Reply:
I'll remove it. Less patch is always better.